app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> str:
+    old_file, new_file = command.split()[1:]
```

Comment:
Also you must validate that command really have 3 parameters and command is "cp"